configs/default/MTKS-MATEKH743.config

```diff
@@ -0,0 +1,191 @@
+resource I2C_SDA 2 B11
+resource LED 1 E03
+resource LED 2 E04
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_SCK 4 E12
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MISO 4 E13
+resource SPI_MOSI 1 D07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource SPI_MOSI 4 E14
+resource ADC_BATT 1 C00
+resource ADC_RSSI 1 C05
+resource ADC_CURR 1 C01
+resource ADC_EXT 1 C04
+resource SDIO_CK 1 C12
+resource SDIO_CMD 1 D02
+resource SDIO_D0 1 C08
+resource SDIO_D1 1 C09
+resource SDIO_D2 1 C10
+resource SDIO_D3 1 C11
+resource PINIO 1 D10
+resource PINIO 2 D11
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 B02
+resource GYRO_EXTI 2 E15
+resource GYRO_CS 1 C15
+resource GYRO_CS 2 E11
+resource USB_DETECT 1 E02
+
+# timer
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer A00 AF2
+# pin A00: TIM5 CH1 (AF2)
+timer A01 AF2
+# pin A01: TIM5 CH2 (AF2)
+timer A02 AF2
+# pin A02: TIM5 CH3 (AF2)
+timer A03 AF2
+# pin A03: TIM5 CH4 (AF2)
+timer D12 AF2
+# pin D12: TIM4 CH1 (AF2)
+timer D13 AF2
+# pin D13: TIM4 CH2 (AF2)
+timer D14 AF2
+# pin D14: TIM4 CH3 (AF2)
+timer D15 AF2
+# pin D15: TIM4 CH4 (AF2)
+timer E05 AF4
+# pin E05: TIM15 CH1 (AF4)
+timer E06 AF4
+# pin E06: TIM15 CH2 (AF4)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer A15 AF1
+# pin A15: TIM2 CH1 (AF1)
+timer C07 AF3
+# pin C07: TIM8 CH2 (AF3)
+timer C06 AF3
+# pin C06: TIM8 CH1 (AF3)
+timer B08 AF1
+# pin B08: TIM16 CH1 (AF1)
+timer B09 AF1
+# pin B09: TIM17 CH1 (AF1)
+
+# dma
+dma ADC 1 8
+# ADC 1: DMA2 Stream 0 Request 9
+dma ADC 3 9
+# ADC 3: DMA2 Stream 1 Request 115
+dma TIMUP 1 0
+# TIMUP 1: DMA1 Stream 0 Request 15
+dma TIMUP 2 0
+# TIMUP 2: DMA1 Stream 0 Request 22
+dma TIMUP 3 2
+# TIMUP 3: DMA1 Stream 2 Request 27
+dma TIMUP 4 1
+# TIMUP 4: DMA1 Stream 1 Request 32
+dma TIMUP 5 0
+# TIMUP 5: DMA1 Stream 0 Request 59
+dma TIMUP 8 0
+# TIMUP 8: DMA1 Stream 0 Request 51
+dma pin B00 0
+# pin B00: DMA1 Stream 0 Request 25
+dma pin B01 1
+# pin B01: DMA1 Stream 1 Request 26
+dma pin A00 2
+# pin A00: DMA1 Stream 2 Request 55
+dma pin A01 3
+# pin A01: DMA1 Stream 3 Request 56
+dma pin A02 4
+# pin A02: DMA1 Stream 4 Request 57
+dma pin A03 5
+# pin A03: DMA1 Stream 5 Request 58
+dma pin D12 6
+# pin D12: DMA1 Stream 6 Request 29
+dma pin D13 7
+# pin D13: DMA1 Stream 7 Request 30
+dma pin D14 12
+# pin D14: DMA2 Stream 4 Request 31
+dma pin E05 0
+# pin E05: DMA1 Stream 0 Request 105
+dma pin A08 14
+# pin A08: DMA2 Stream 6 Request 11
+dma pin A15 0
+# pin A15: DMA1 Stream 0 Request 18
+dma pin C07 0
+# pin C07: DMA1 Stream 0 Request 48
+dma pin C06 0
+# pin C06: DMA1 Stream 0 Request 47
+dma pin B08 0
+# pin B08: DMA1 Stream 0 Request 109
+dma pin B09 0
+# pin B09: DMA1 Stream 0 Request 111
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature LED_STRIP
+feature OSD
+
+# serial
+serial 5 64 115200 57600 0 115200
+serial 7 1024 115200 57600 0 115200
```

Comment:
remove serial 7 definition, leave it as default

Reply:
This is the ESC_SENSOR input. See http://www.mateksys.com/wp-content/uploads/2020/10/H743-SLIM_Wiring.jpg